beta-src/src/components/ui/WDCountryTable.tsx

```diff
@@ -0,0 +1,177 @@
+import * as React from "react";
+import {
+  Box,
+  Chip,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  tableCellClasses,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+import BetIcon from "./icons/country-table/WDBet";
+import CentersIcon from "./icons/country-table/WDCenters";
+import { CountryTableData } from "../../interfaces";
+import DelaysIcon from "./icons/country-table/WDDelays";
+import Device from "../../enums/Device";
+import IntegerRange from "../../types/IntegerRange";
+import PowerIcon from "./icons/country-table/WDPower";
+import UnitsIcon from "./icons/country-table/WDUnits";
+
+interface WDCountryTableProps {
+  countries: CountryTableData[];
+  device: Device;
+  maxDelays: IntegerRange<0, 5>;
+}
+
+interface Column {
+  align?: "right" | "left" | "center";
+  icon?: React.FC;
+  id: keyof CountryTableData;
+  label: string;
+}
+
+const columns: readonly Column[] = [
+  { id: "power", label: "Power", icon: PowerIcon, align: "left" },
+  { id: "unitQty", label: "Units", icon: UnitsIcon, align: "center" },
+  {
+    id: "centerQty",
+    label: "Centers",
+    icon: CentersIcon,
+    align: "center",
+  },
+  {
+    id: "bet",
+    label: "Bet",
+    icon: BetIcon,
+    align: "center",
+  },
+  {
+    id: "delaysLeft",
+    label: "Delays",
+    icon: DelaysIcon,
+    align: "center",
+  },
+];
+
+const WDCountryTable: React.FC<WDCountryTableProps> = function ({
+  countries,
+  device,
+  maxDelays,
+}): React.ReactElement {
+  const mobileLandscapeLayout =
+    device === Device.MOBILE_LANDSCAPE || device === Device.MOBILE_LG_LANDSCAPE;
+  const WDTableCell = styled(TableCell)(() => {
+    const padding = mobileLandscapeLayout ? "6px 6px" : "6px 16px";
+    return {
+      [`&.${tableCellClasses.head}`]: {
+        lineHeight: "unset",
```

Comment:
why unset? let's make it an actual value. maybe 1 or normal?

Reply:
I have no problem changing this, but what's wrong with unset? I want to unset the MUI rule.

Reply:
i think an override to an actual value for line-height is better than leaving us exposed to an inheritance issue.